main/ertp/guide/gotchas.md

```diff
@@ -0,0 +1,23 @@
+# Gotchas
+
+When looking at the code in our [tests](https://github.com/Agoric/ERTP/tree/master/test), you might see some new
+concepts:
+
+* __Vats__: All user code runs in what we call a `vat`. Within a vat,
+  code is run synchronously. Communication with another vat happens
+  asynchronously. The [SwingSet
+  infrastructure](https://github.com/Agoric/SwingSet) creates the vats
+  and makes communication between vats possible.
+
+* __E() and infix bang (!)__: Instead of `obj.foo()`, we can write
```

Comment:
I believe this is old syntax and is now wavy dot ` ~.`

@katelynsills can you confirm that?

Reply:
Yes, you're right. I had updated this text on the ERTP github but missed it here https://github.com/Agoric/ERTP/blob/master/README.md